packages/p2p-v2/src/pages/my-ads/screens/MyAds/MyAdsTable/MyAdsTable.tsx

```diff
@@ -46,13 +47,48 @@ const columns = [
     },
 ];
 
+type TMyAdsDisplayWrapperProps = {
+    isPaused: boolean;
+    onClickToggle: () => void;
+};
+
+const MyAdsDisplayWrapper = ({ children, isPaused, onClickToggle }: PropsWithChildren<TMyAdsDisplayWrapperProps>) => {
```

Comment:
what is this component for?

Reply:
for rendering the header and footer in mobile and desktop separately i created a wrapper which has fullpagemobilewrapper.